### DISCUSSION TITLE
PCAPs stuck at pending on new environment

### DISCUSSION THREAD

**Discussion by Hammer-Bear** [Security-Onion-Solutions/securityonion#5026] [Unsupported Versions]
Hey all,

So we built a new SO 2.3.61 environment last week in a distributed deployment. We let the forward collect some data over the weekend and when pivoting from alerts into PCAP the job queues but sits on "Pending" had a look in the sensoroni.log file on the sensor and below is what is coming back. The host files of the sensor have the internal IP of the manager node and the search nodes so not sure why it's timing out when trying to request.

`timestamp=2021-08-02T10:10:54.706912743Z level=warn message="Failed to submit request" error="Post \"https://(hostname)/sensoroniagents/api/node\": dial tcp (exnternal address):443: connect: connection timed out"`

**Comment by TOoSmOotH** [comment]
Looks like the sensor cannot hit the manager on port 443.

**Reply by Hammer-Bear** [reply]
Assuming this would be firewall related then? i did have a look through it and it looked fine but i also wasnt the one that built the sensor as it was built while i was away, i will check with them in the morning and double check the firewall rules between the manager and sensor.
